assets/js/index.js

```diff
@@ -0,0 +1,97 @@
+let library = setupLibrary();
```

Comment:
We aren't, in this scope, reassigning `library` to any other value, aren't we? In that case i'd prefer us to use `const` instead of `let`.